README.md

```diff
@@ -1,22 +1,35 @@
-Unicode Blocks
-====================
+# `unicode-blocks`
 
 [![CI](https://github.com/magiclen/unicode-blocks/actions/workflows/ci.yml/badge.svg)](https://github.com/magiclen/unicode-blocks/actions/workflows/ci.yml)
+[![Documentation](https://docs.rs/unicode-blocks/badge.svg)](https://docs.rs/unicode-blocks)
```

Comment:
I'm not used to using this badge.

Reply:
> I'm not used to using this badge.
Most users are though.

If you look at the average popular crate updated in the last two years on `crates.io`, you'll probably see badges for build/CI status, `crates.io` version and `docs.rs` link in the README.

The latter two are for people coming to/accrorss the crate via the repo and not via crates.io. I.e. they will have to dig to find out if this is a published crate at all and if there are docs (the former does not imply the latter, some crates have shoddy docs or the `docs.rs` build fails).

Reply:
Fair enough, I will think about it.

Reply:
I don't know if shields.io is trustworthy. Displaying their badges might be promoting their services.

Reply:
Good point.
It's seem what most Rust crate repos on GH seem to use.
I used something else two years back (forgot the name) but that stopped working some time ago so I switched to `shields.io` just copying what I saw in others's READMEs.